packages/core/docs/guide/getters.md

```diff
@@ -0,0 +1,171 @@
+# Getters
+
+## What are getters?
+
+Getters are pure functions which allows you to fetch response from integrated ecommerce platform. They return an agnostic or primitive type. All getters are grouped into domains they are associated with e.g. `cartGetters` and named according to values they return e.g. `getCartTotalItems`. 
+
+
+## How can you create and use getters?
```

Comment:
This is for integrators. I don't think regular users need that information

Reply:
Ok removed.